src/App.js

```diff
@@ -1,30 +1,54 @@
-import { useEffect, useState } from "react"
-import { Link, Route, Routes } from "react-router-dom"
-import ContactsList from "./components/ContactsList"
-import ContactsAdd from "./components/ContactsAdd"
-import ContactsView from "./components/ContactsView"
-import "./styles/styles.css"
+import { useEffect, useState } from "react";
+import { Link, Route, Routes } from "react-router-dom";
+import ContactsList from "./components/ContactsList";
+import ContactsAdd from "./components/ContactsAdd";
+import ContactsView from "./components/ContactsView";
```

Comment:
When there are too many imports, I like to group them by leaving a brief description like so:
``` // Component
import ContactsList from "./components/ContactsList";
import ContactsAdd from "./components/ContactsAdd"
import ContactsView from "./components/ContactsView" ```